.github/workflows/test.yml

```diff
@@ -0,0 +1,40 @@
+name: Test
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+          - pypy3
+        tox-environment:
+          - django22-alchemy-mongoengine
+          - django30-alchemy-mongoengine
+          - django31-alchemy-mongoengine
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.3.0
```

Comment:
I am not familiar with the actions repository; how did you choose this action?

Reply:
It's the most commonly use as for now: https://github.com/marketplace?type=actions&query=mongodb

MongoDB Inc didn't feel the need to support official one until now.

It's not super great, but the risks are limited and it is the best option for now.

Maybe add a todo to check that in the future?

Reply:
I chose this one as it seemed to be the most popular and common choice from my limited research.

If you have a preference for a different approach, please let me know and I'll take a look.

Reply:
Based on [this issue](https://github.com/actions/example-services/issues/8#issuecomment-559484139), I believe it should be possible with:

```yaml
services:
  mongodb:
    image: mongo:latest
    ports:
      - 27017:27017
```

(And that's basically what I use locally for development, too :wink:)

Reply:
Nice, this worked! Thanks for pointing this out.

I've added this in the latest version. You can see it working here:

https://github.com/jdufresne/factory_boy/actions/runs/373502935